File: Web Alex Roel/s7-JS-Estructuras-Array/switch.js
let letra = 'A';

switch(letra){
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
    case 'A':
    case 'E':
    case 'I':
    case 'O':
    case 'U':
        console.log(`La ${letra} es una vocal`);
        break;
    default:
        console.log(`La ${letra} no es una vocal`);
        break;
}

File: Web Alex Roel/s33-Go Control/switch.go
package main

import "fmt"

func main() {
	var vocal string
	fmt.Print("Ingrese una letra: ")
	fmt.Scanln(&vocal)

	//Switch
	/*switch {
	case vocal == "a" || vocal == "A":
		fmt.Println(vocal, " es vocal.")
	case vocal == "e" || vocal == "E":
		fmt.Println(vocal, " es vocal.")
	case vocal == "i" || vocal == "I":
		fmt.Println(vocal, " es vocal.")
	case vocal == "o" || vocal == "O":
		fmt.Println(vocal, " es vocal.")
	case vocal == "u" || vocal == "U":
		fmt.Println(vocal, " es vocal.")
	default:
		fmt.Println(vocal, " no es vocal.")
	}
	*/
	//Switch con multiples casos con la misma respuesta

	switch vocal {
	case "a", "A", "e", "E", "i", "I", "o", "O", "u", "U":
		fmt.Println(vocal, " es vocal.")
	default:
		fmt.Println(vocal, " no es vocal.")
	}
}
